### PR TITLE
SOC-745 Changing hero unit for /join page

### PR DESCRIPTION
**Pull request by bkoval**
@mklucsarits https://wikia-inc.atlassian.net/browse/SOC-745

### CODE REVIEW

File: front/locales/en/auth.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	},
 	"join": {
 		"title": "Join Wikia",
-		"register-with": "Register with",
+		"join-wikia-today": "Join Wikia Today",
```
Comment (bkoval) [inline front/locales/en/auth.json]: Any idea on a better name for this thing? ;)

Comment (aquilax) [inline front/locales/en/auth.json]: How about `join-text`?

Comment (mklucsarits) [inline front/locales/en/auth.json]: I agree with our guest star commenter.

Comment (bkoval) [inline front/locales/en/auth.json]: Wowow, So glad to hear from you @aquilax !!! :dancer:

**Pull request closed without merging**